src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -69,7 +69,19 @@
    :ruby-load-path           [schema/Str]
    :gem-path                 schema/Str
    :signeddir                schema/Str
-   :serial                   schema/Str})
+   :serial                   schema/Str
+   ;; This would be list of compile master node names (FQDNs) including MoM
+   ;; It would either be provisioned by PE or created & maintained by user (input configuration)
+   :compile-masters          [schema/Str]    
+   ;; This would point to a file containing a list of serial numbers of compile master certificates
```

Comment:
If you're going to add comments, and a little bit is good, I would make them more declarative, like "path to file containing serials found for certs listed in `compile-masters`".